tests/testapps/CTCatalog/CTCatalog-WinStore10.sln

```diff
@@ -0,0 +1,43 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I was lazy and I tried to use this sample to verify the logic but I am not able to load this project. It keeps crashing VS